rust/azure_iot_operations_connector/src/lib.rs

```diff
@@ -4,7 +4,24 @@
 //! Connector framework for Azure IoT Operations
 
 #![warn(missing_docs)]
+
+use azure_iot_operations_services::schema_registry::PutRequest;
+
 pub mod base_connector;
+pub mod data_transformer;
 pub mod destination_endpoint;
 pub mod filemount;
 pub mod source_endpoint;
+
+/// Message Schema to send to the Schema Registry Service
+pub type MessageSchema = PutRequest;
+
+/// Struct format for data sent to the [`DataTransformer`] and the destination
+pub struct Data {
```

Comment:
Do we want to go ahead and add a HLC now, or wait until someone asks for it? Though I think there's a good chance that someone will be me. :)

Reply:
good catch - I'll add it now, but with a comment that we might remove it. Easier to unplug things later than to try and fit it in later

Reply:
added - figured this would need to be Option, but it might not end up needing to be that way